### PR TITLE
Add translation domains for modules messages

### PR DESCRIPTION
**Pull request by AlexEven**
| Questions | Answers |
| --- | --- |
| Branch? | develop |
| Description? | Add domains to all modules page - part 1 |
| Type? | N/A |
| Category? | BO |
| BC breaks? | no |
| Deprecations? | no |

### CODE REVIEW

File: src/PrestaShopBundle/Controller/Admin/ModuleController.php [hunk 1]
```diff
@@ -191,30 +191,30 @@ public function moduleAction(Request $request)
                 if ($response[$module]['status'] === null) {
                     $response[$module]['status'] = false;
                     $response[$module]['msg'] = $translator->trans(
-                        "%s.' did not return a valid response on %s action",
+                        '%s did not return a valid response on %s action.',
                         array($module, $action),
```
Comment (maximebiloe) [inline src/PrestaShopBundle/Controller/Admin/ModuleController.php]: Don't you need to name your variables to have `trans()` working?

File: src/PrestaShopBundle/Controller/Admin/ModuleController.php [hunk 2]
```diff
@@ -191,30 +191,30 @@ public function moduleAction(Request $request)
...
-                        'AdminModules'
+                        'Admin.Notifications.Error'
                         );
                 } elseif ($response[$module]['status'] === false) {
                     $error = $moduleManager->getError($module);
                     $response[$module]['msg'] = $translator->trans(
                         'Cannot %s module %s. %s',
                         array(str_replace('_', ' ', $action), $module, $error),
```
Comment (maximebiloe) [inline src/PrestaShopBundle/Controller/Admin/ModuleController.php]: Same here

File: src/PrestaShopBundle/Controller/Admin/ModuleController.php [hunk 3]
```diff
@@ -191,30 +191,30 @@ public function moduleAction(Request $request)
...
-                        'AdminModules'
+                        'Admin.Notifications.Error'
                     );
                 } else {
                     $response[$module]['msg'] = $translator->trans(
                         '%s action on module %s succeeded.',
                         array(ucfirst(str_replace('_', ' ', $action)), $module),
```
Comment (maximebiloe) [inline src/PrestaShopBundle/Controller/Admin/ModuleController.php]: Same here

File: src/PrestaShopBundle/Controller/Admin/ModuleController.php (hunk 1 above)
Comment (Quetzacoalt91) [inline src/PrestaShopBundle/Controller/Admin/ModuleController.php]: Leave it with me, another PR will be created for this task.

File: src/PrestaShopBundle/Controller/Admin/ModuleController.php (hunk 2 above)
Comment (Quetzacoalt91) [inline src/PrestaShopBundle/Controller/Admin/ModuleController.php]: And same answer

File: src/PrestaShopBundle/Controller/Admin/ModuleController.php (hunk 3 above)
Comment (Quetzacoalt91) [inline src/PrestaShopBundle/Controller/Admin/ModuleController.php]: And same answer

**Pull request merged**